### ISSUE TITLE
No Thumbnails Displayed in Detail View

### ISSUE THREAD

**Issue by fraxgut** [nomacs/nomacs#1152]
**Describe the Bug**  
When using the detail view in Windows Explorer, no thumbnail previews are displayed for any image format, including common ones like JPEG. This makes it difficult to identify images based on their content.

**To Reproduce**  
Steps to replicate the issue:
1. Open a directory containing image files in various formats (e.g., JPEG, PNG, GIF, AVIF, JXL).
2. Switch to the detail view in Windows Explorer.
3. Notice that no thumbnails are displayed for any of the image files, regardless of their format.

**Expected Behavior**  
Thumbnail previews should be visible for all supported image formats in the detail view, enabling easy identification of images without opening them.

**Screenshots**  
![imagen](https://github.com/user-attachments/assets/752e85d6-3adc-4f08-8d5a-f87cf12ce383)

**Desktop Setup:**
- **OS:** Windows 10 Enterprise LTSC IoT 21H2
- **Version:** 3.19.0 Testing (Pre-release)

**Additional Context**  
Classical theme in use, indicating no Visual Style (.msstyle) is active.

**Comment by scrubbbbs** [comment]
This doesn't look like a screenshot of nomacs. I'm not sure if nomacs could cause this to happen. If you uninstall nomacs do you get previews back?

**Comment by fraxgut** [comment]
> This doesn't look like a screenshot of nomacs. I'm not sure if nomacs could cause this to happen. If you uninstall nomacs do you get previews back?

I just realized I posted the wrong screenshot. I’ve edited the original post to show the correct picture.

Here’s how it looks after replacing Nomacs with IrfanView for testing:
![imagen](https://github.com/user-attachments/assets/093d287c-2bc8-4788-b3b2-cbf490a8b257)

It seems the issue persists even in Detail mode with IrfanView, so maybe it’s a Windows issue.
